src/config.js

```diff
@@ -365,16 +377,42 @@ export default class Config {
     } else {
       this._cacheRootFolder = String(cacheRootFolder);
     }
+
+    const manifest = await this.maybeReadManifest(this.cwd);
+
+    const plugnplayByEnv = this.getOption('plugnplay-override');
+    if (process.platform === 'win32') {
```

Comment:
Yep, totally! Windows support is to be implemented, but right now my schedule is tight so I'd appreciate if someone from the community could lend a hand. If I remember correctly, the main blocker simply is figuring out how to cleanly convert Windows paths into Posix paths, back and forth (since we now use relative paths it should be easier - won't have to deal with device letter anymore).